lib/reversetunnel/transport.go

```diff
@@ -298,12 +298,11 @@ func (p *transport) start() {
 		errorCh <- err
 	}()
 
+	// wait for both io.Copy goroutines to finish, or for
+	// the context to be canceled.
 	for i := 0; i < 2; i++ {
 		select {
-		case err := <-errorCh:
-			if err != nil && err != io.EOF {
-				p.log.Warnf("Proxy transport failed: %v %T.", trace.DebugReport(err), err)
-			}
+		case <-errorCh:
```

Comment:
Looks like before we were reporting errors other than `io.EOF`. What errors were you seeing here other than `io.EOF` that were not relevant?

Reply:
Removed because this was just filling the logs with errors related to "use of closed network connection".